README.md

```diff
@@ -1,2 +1,188 @@
+Enter Spacelift endpoint (eg. https://unicorn.app.spacelift.io/):
+```
+
+In the next step, you will be asked to choose which authentication method you are going to use. Note that if your account is using [SAML-based SSO authentication](https://docs.spacelift.io/integrations/single-sign-on), then API keys are your only option. After you're done entering credentials, the CLI will validate them against the server, and assuming that they're valid, will persist them in a credentials file in `.spacelift/${MY_ALIAS}`. It will also create a symlink in `${HOME}/.spacelift/current` pointing to the current profile.
+
+You can switch between account profiles by using `spacelift-cli account select ${MY_ALIAS}`. What this does behind the scenes is point `${HOME}/.spacelift/current` to the new location. You can also delete stored credetials for a given profile by using the `spacelift-cli account logout ${MY_ALIAS}` command.
+
+## Usage
+
+Currently only a small subset of operations is supported, starting with the ability to trigger runs and tasks and see their logs.
+
+### Run management
+
+Managing runs is currently avaialble through the `stack` subcommand:
+
+```bash
+❯ spacelift-cli stack --id=stack-id help
+NAME:
+   spacelift-cli stack - Manage a Spacelift stack
+
+USAGE:
+   spacelift-cli stack [global options] command [command options] [arguments...]
+
+COMMANDS:
+   help, h  Shows a list of commands or help for one command
+   Run management:
+     deploy   Start a deployment (tracked run)
+     logs     Show logs for a particular run
+     preview  Start a preview (proposed run)
+     task     Perform a task in a workspace
+
+GLOBAL OPTIONS:
+   --id value  User-facing ID (slug) of the stack
+   --help, -h  show help (default: false)
+```
+
+Each of these operations will require user-facing stack ID (so called "slug") visible - among other places - in the stack URL in the GUI. Let's look at the operations one by one.
+
+#### `stack deploy` subcommand
+
+```bash
+❯ spacelift-cli stack --id=stack-id deploy --help
+NAME:
+   spacelift-cli stack deploy - Start a deployment (tracked run)
+
+USAGE:
+   spacelift-cli stack deploy [command options] [arguments...]
+
+CATEGORY:
+   Run management
+
+OPTIONS:
+   --sha value  Commit SHA for the newly created run
+   --tail       Indicate whether to tail the run (default: false)
+   --help, -h   show help (default: false)
+```
+
+This subcommand creates a [tracked run](https://docs.spacelift.io/concepts/run/tracked) on a stack. By default, it uses the latest tracked commit, but you can also point it to an arbitrary SHA using the `--sha` flag. Also by default, this command only creates a run and prints out the URL where the run can be accessed. If the `--tail` flag is set however, run logs will be retrieved until the run terminates. If the run is tailed, the exit code of the command will depend on the outcome of the run - if the run is `FINISHED`, the command succeeds (exits with `0`), otherwise it fails.
```

Comment:
This is maybe a wider question than just for the CLI, but would it make sense to stick to one name (i.e. either _deployment_ or _tracked run_)?

While both names are used, it definitely makes sense to explain it here like you've done, so I'm not suggesting you change this text just now. I just bring it up because it feels slightly weird to `spacelift-cli stack deploy` if we then say it creates a "tracked run" rather than saying it "deploys the stack".

Reply:
It would. I like the word `deployment` better but historically we called them `tracked runs`, and that's passed to policies which folks may have already written, so I'm trying to steer us gently towards what I see as a the better (more universal) name.

You know, naming things is hard, especially in a foreign language.

Reply:
Yeah, and deprecating things is really tough as well, so I'm definitely not criticising here - just wondered.